BooruSharp/Booru/ABooru.cs

```diff
@@ -301,10 +294,11 @@ protected get
         protected static readonly Random _random = new Random();
         // TODO: remove this message after removing obsolete constructors.
         protected const string _deprecationMessage = "Use a contructor that accepts single BooruOptions parameter. Use | (bitwise OR) operator to combine multiple options.";
+        private const string _userAgentHeaderValue = "Mozilla/5.0 BooruSharp";
```

Comment:
This should be replaced with something sensible like `"BooruSharp/1.2.3"` in the future (I have some logic that takes care of pulling assembly version in mind), at least according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#Syntax). After all, we aren't using anything Mozilla related in this project.

Reply:
I think we can change that without any risk of breaking things